lib/uo_cli.rb

```diff
@@ -406,19 +399,36 @@ def self.run_func
 
       feature_file = URBANopt::GeoJSON::GeoFile.from_file(featurefile)
       if @opthash.subopts[:reopt] == true || @opthash.subopts[:reopt_scenario] == true || @opthash.subopts[:reopt_feature] == true
+        reopt_files_dir = File.join(@root_dir, 'reopt/')
+        create_reopt_scenario_file(@opthash.subopts[:scenario])
```

Comment:
is this a "just in case" call to create_reopt_scenario_file?  (the csv *should have the reopt columns if it was made with this newest code, but in case it does not, the column will be added before postprocess occurs?)

Reply:
Exactly.

Reply:
And I fixed a dumb dumb dumb bug I created when I first wrote that `run_func`. We are still assuming that all buildings in the scenario use the same assumptions file - is that a valid assumption?

Reply:
it's a valid default. but users might modify those values in the csv file before they post-process it.

Reply:
@kflemin Unless I'm mistaken the assumptions file is set when we instantiate the post-processor, which only happens once per scenario (not once per building). To send multiple assumptions files to reopt we'll need to change the way the cli calls the post-processor.